gcloud/bigtable/happybase/pool.py

```diff
@@ -27,6 +28,14 @@
 """Minimum allowable size of a connection pool."""
 
 
+class NoConnectionsAvailable(RuntimeError):
```

Comment:
In this case it's part of the public surface of HappyBase. I can implement it in `gcloud.exceptions` and import it here, but I don't feel like there is a payoff there.